File: procesamiento3.m



n=-50:50;
x=cos(pi*0.1*n);
y=cos(pi*0.9*n);
z=cos(pi*2.1*n);
subplot(3,1,1)
plot(n,x)
title('x[n]=cos(0.1\pin')
grid
subplot(3,1,2)
plot(n,y)
title('y[n]=cos(0.9\pin)')
grid
subplot(3,1,3)
plot(n,z)
grid
title('z[n]=cos(2.1\pin)')
xlabel('n')

File: procesamiento3.py
import numpy as np
import matplotlib.pyplot as plt

n = np.arange(-50, 51)
x = np.cos(np.pi * 0.1 * n)
y = np.cos(np.pi * 0.9 * n)
z = np.cos(np.pi * 2.1 * n)

plt.subplot(3, 1, 1)
plt.plot(n, x)
plt.title(r'$x[n]=\cos(0.1\pi n)$')
plt.grid()

plt.subplot(3, 1, 2)
plt.plot(n, y)
plt.title(r'$y[n]=\cos(0.9\pi n)$')
plt.grid()

plt.subplot(3, 1, 3)
plt.plot(n, z)
plt.title(r'$z[n]=\cos(2.1\pi n)$')
plt.grid()

plt.xlabel('n')

plt.show()
